Fixed code formatting of the NextVisibleTime Type (functions-bindings-storage-queue.md)

gjlumsden opened the pull request:
Fixed the code formatting for the type of NextVisibleTime so that a closing back-tick is included and the type name is correctly formatted with syntax highlighting.

azurecla commented:
Hi __@damienpontifex__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

davidwrede commented:
Hi damienpontifex,

Great catch! Azure continues to update at a rapid pace, so thank you for helping to make our documentation current and up-to-date by submitting PRs like this.

Thanks,
Dave

damienpontifex commented:
@davidwrede I would assume this is also a valid change for the other client platforms as the backend component is identical. Would you like me to do a separate PR for these or all of this be handled as the update is now recognised?

gjlumsden committed:
Fixed code formatting of the NextVisibleTime Type

Changed the code formatting for the type of NextVisibleTime so that a closing back-tick is included.
